src/app/controllers/followerships_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FollowershipsController < ApplicationController
+  before_action :set_user
+
+  def create
+    current_user.follow(@user)
```

Comment:
I think we'll need to remove the `follow` and `unfollow` methods from the model and create the record directly here, as we didn't use this method but here we don't need to abstract it away, also we'll need to handle creation error using flash message.